git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
Is it possible to trace back a commit to the PR that it belonged to? On CS projects (especially older ones), I find it difficult to find the reason for a change unless the ticket # is mentioned in the git commit message (or a comment in the file, which is rare). Having the ticket # right there can help you find the JIRA ticket/PR easily.